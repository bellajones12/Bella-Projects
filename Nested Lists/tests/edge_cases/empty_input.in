NEW 1

TYPE 0