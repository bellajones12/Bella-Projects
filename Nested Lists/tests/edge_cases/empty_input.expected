List 0: 
string
